### ISSUE TITLE
What's the result of EM and F1 you finally got?

### ISSUE THREAD

**Issue by zhenglm** [MurtyShikhar/Question-Answering#2]

**Comment by MurtyShikhar** [comment]
Around 60. You'll have to tune dropout and learning rates. I don't remember the values I used but they were around 0.25 dropout and around 0.0005 learning rate.

**Comment by zhenglm** [comment]
OK , I'll try . Thanks!

**Comment by nikefd** [comment]
hi, @MurtyShikhar , I can't find the learning_rate in config.py, where can I find it. Can you help me? Thank you very much!

**Comment by nikefd** [comment]
I understand, it is in the file `qa_model.py` -> `tf.train.AdamOptimizer()`, so you just use the default learning_rate ?
